### DISCUSSION TITLE
Automatic Game Scanning for Ubisoft Connect?

### DISCUSSION THREAD

**Discussion by BakaKuma-uwu** [moraroy/NonSteamLaunchers-On-Steam-Deck#165] [Q&A] [answered]
Hey, I installed the latest version, imported my Assassin's Creed Odyssey as well as Prince of Persia files, and the games don't appear in my Steam library. Is it because I installed them on my SD card?

**Comment by moraroy** [comment]
@RobinNyan very true actually I think I only have it set for the default location  C: Drive ... the automatic scanner was looking in there

**Reply by BakaKuma-uwu** [reply]
Heh good to know :D I still love the program though, as it allows me to install third party games to my SD card :)

**Reply by moraroy** [reply]
@RobinNyan in the upcoming days I'll try to update it so it's a little better at find the ubi games, for now it should work at the default location I hope lol

**Reply by BakaKuma-uwu** [reply]
don't stress yourself too much!

**Reply by atrenjes** [reply]
The automatic game scanning seems very broken. I can't get Ubisoft games to work and I can't launch Epic games that have been automatically scanned. Every time I reset my deck, I end up with an extra shortcut added for the game in the non-steam games section, but without icons.

**Reply by moraroy** [reply]
@atrenjes yeah for some reason.. that's happening for some people...the ubisoft games Scanner isn't working right in general,  but I'm still trying to figure out why there's a duplication going on

**Reply by moraroy** [reply, marked as answer]
@RobinNyan @atrenjes I updated the script to correctly parse the ubi soft Games now. Let me know how it goes still trying to figure out why it's duplicating though!
